usergroups.go

```diff
@@ -50,20 +51,45 @@ func (api *Client) userGroupRequest(ctx context.Context, path string, values url
 	return response, response.Err()
 }
 
+// CreateUserGroupParams contains arguments for CreateUserGroup method call
+type CreateUserGroupParams struct {
```

Comment:
Can we make this private and same with `EnableSection` to keep it consistent with the pattern we already have? 

See [chat.go](https://github.com/slack-go/slack/blob/34f9a10b71d70bdcd1b9456a6d91805aca8000bf/chat.go#L398).

Reply:
done.